### PR TITLE
changed all vars, added error to callback

### PR DESCRIPTION
**Pull request by edkotkas**

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var vetus = require('./app')
+let vetus = require('./app')
```
Comment (jmsv) [inline example.js:1]: const?

File: app.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-var path = require('path')
-var fs = require('fs')
+let path = require('path')
```
Comment (jmsv) [inline app.js]: const?

File: app.js [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+let fs = require('fs')
```
Comment (jmsv) [inline app.js]: const?

File: app.js [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 
-var Collection = require('./src/collection')
+let Collection = require('./src/collection')
```
Comment (jmsv) [inline app.js]: const?

File: src/collection.js [hunk 5]
```diff
@@ -1,33 +1,38 @@
-var path = require('path')
-var Repository = require('./repository')
-var fs = require('fs')
-var Promise = require('promise')
-var mkdirp = require('mkdirp')
+let path = require('path')
```
Comment (jmsv) [inline src/collection.js]: const? for alla these

**Review by jmsv — CHANGES_REQUESTED** [review]

**Comment by jmsv** [comment]
`const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const const`

File: src/collection.js [hunk 6]
```diff
@@ -39,12 +44,12 @@ module.exports = function(options) {
 
           collection.data = data
           callback()
-        })
+        }).catch(callback)
```
Comment (envman) [inline src/collection.js]: doh

**Review by jmsv — APPROVED** [review]

**Review by envman — APPROVED** [review]

**Pull request merged**